Point center

Point half_size
